pulp_2to3_migration/app/migration.py

```diff
@@ -88,11 +87,11 @@ def migrate_repositories(plan):
                 repo_version_setup = repos_to_create[pulp3_repo_name].get('repository_versions')
                 for repo_version in repo_version_setup:
                     pulp2_repo_ids.append(repo_version['repo_id'])
-                pulp2repos_qs = Pulp2Repository.objects.filter(pulp2_repo_id__in=pulp2_repo_ids)
-                for pulp2repo in pulp2repos_qs:
-                    if not pulp2repo.pulp3_repository:
-                        pulp2repo.pulp3_repository = repo
-                        pulp2repo.save()
+                pulp2repos_qs = Pulp2Repository.objects.filter(
+                    pulp2_repo_id__in=pulp2_repo_ids, pulp3_repository__isnull=True
+                )
+                pulp2repos_qs.update(pulp3_repository=repo)
+
```

Comment:
This part is actually not a removal, just a small optimization to the existing code.
It somehow got sneaked into this PR :eyes: :innocent: